docs/application/web/api/5.0/ui_fw_api/Mobile_UIComponents/mobile_component_list.htm

```diff
@@ -7,7 +7,7 @@
 	<link href="../snippet.css" rel="StyleSheet" type="text/css">
 	<title>Mobile UI Components</title>
 </head>
-<body onload="prettyPrint()" marginwidth="0" marginheight="0" id="content">
```

Comment:
@dorlinski 
Could you remove unnecessary attributes `marginwidth="0" marginheight="0"` ? Thank you.

Reply:
Fixed